VideoWeb/VideoWeb.AcceptanceTests/Features/Filters.feature

```diff
@@ -12,7 +12,7 @@ Scenario: VHO filters hearings by alert type
   When the user filters by alert with the options Disconnected,Self-test failed,Cam/mic blocked,Suspended
   Then the hearings are filtered
 
-@VIH-5846 @Smoketest-Extended
+@VIH-5846 @Smoketest-Extended  @Ignore
 Scenario: VHO filters hearings by location
```

Comment:
Is this functionality permanent? In which case just delete the test rather than ignore it

Reply:
there are a couple of more marked as ignore, leaving this for now. Not sure if we need it .

Reply:
okay np